src/components/BarList.js

```diff
@@ -0,0 +1,23 @@
+import classes from './BarList.module.css'
+import Bar from './Bar'
+
+const BarList = (props) => {
+  return (
+    <ul className={classes['bar-list']}>
+      {props.bars.map((bar) => (
+        <Bar
+          key={bar.id} 
+          name={bar.name} 
+          street={bar.street} 
+          city={bar.city} 
+          state={bar.state} 
+          zipcode={bar.postal_code} 
+          phone={bar.phone} 
+          site={bar.website_url}
```

Comment:
Does this need to be broken down to this granular level, or could you just pass the bar object in and let the bar component handle the props?

```suggestion
          bar={bar}
```

And then in the bar component:
```js
const Bar = (props) => {
  const { bar } = props;
  return (
    <li className={classes.bar}>
      <h2>{bar.name}</h2>
      <h3>{bar.street}</h3>
      <h3>{bar.city}</h3>
      <h3>{bar.state}</h3>
      <h3>{bar.postal_code}</h3>
      <h3>{bar.phone}</h3>
      <h3>{bar.website_url}</h3>
    </li>
  )
}
```